### PR TITLE
Hamburger-Menu:

### PR DESCRIPTION
**Pull request by sam2if**
In this project:

1) index.html file is added for the webpage
2) style.css file is added for the stylesheet
3) hamburgers.js file is added to display the menu
4) readme.md file is added to explain the project.
5) linters file is added to fix errors.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,20 +15,25 @@
...
+              <li>
+                <a href="#work-section" class="navnav-items" style="text-decoration: none;">Portfolio</a>
+              </li>
+              <li>
+                <a href="#about-section" class="navnav-item" style="text-decoration: none;">About</a>
+              </li>
+              <li>
+                <a href="#contact-section" class="navnav-item" style="text-decoration: none;">Contact</a>
+              </li>
+            </ul>
```
Comment (DuaneDave) [inline index.html:31]: Currently, when I run your project, the hamburger menu is showing which is not the expectation according to the Figma template. Kindly make it disappear and let the links be visible on the desktop and the hamburger on the mobile in order to be consistent with Figma.

Your Header

Badheading

FIGMA

Goodheading

File: index.html [hunk 2]
```diff
@@ -15,20 +15,25 @@
...
 
-            <div class="menu-bar">
-              <img src="./images/menu-bar.png" alt="menu-bar" />
+            <div class="hamburger">
+              <span class="barbar"></span>
+              <span class="barbar"></span>
+              <span class="barbar"></span>
             </div>
           </nav>
```
Comment (DuaneDave) [inline index.html:38]: The styling for the menu is currently not consistent with the Figma design. Please fix the space issue (between the close icon and the nav links and between the top of the blue box and the nav links) and colour for the close button in order to be consistent with the Figma design.

YOUR DESIGN

navSpace


FIGMA

betterNavSpace

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @sam2if and @KAYEMBAJONATHAN  👋 , 

Good job so far on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ The app is working as expected
✔️ Good usage es6 syntax
✔️ PR summary present


## Required Changes ♻️
- One of the requirements of this milestone, is to learn to set up a JavaScript linter in your project. Please add Js to the set-up in order to learn this in preparation for module 2. Check out the [config](https://github.com/microverseinc/linters-config/tree/master/html-css-js) here.

lints


_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @sam2if 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- You asked for the changes in a polite and friendly way.
- Your linters are passing all the checks.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**